docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -169,6 +169,18 @@ GF_AUTH_AZUREAD_CLIENT_SECRET
 
 **Note:** Verify that the Grafana [root_url]({{< relref "../../../configure-grafana/#root-url" >}}) is set in your Azure Application Redirect URLs.
 
+### Access token expiration check and configure refresh token
```

Comment:
I would simply go with ### Configure refresh token

The reason: You address the access token expiration check in the intro text that follows and this section is really about the task of configuring a refresh token.

Reply:
Done, thanks!